samples/drivers/virtualization/ivshmem/doorbell/README.rst

```diff
@@ -69,7 +69,7 @@ How to
    Due to limited RAM memory available in qemu_x86_64 dts, it is not possible
    to use the default shared memory size of ivshmem (4MB) for qemu_x86_64.
 
-Steps to reproduce this test:
+Steps to reproduce this sample:
 
 - Run ivshmem-server. If vectors != 2, you need to change ivshmem driver
```

Comment:
It would be better if you give a code block here to show how you run ivshmem-server. For example:

```
$ sudo ivshmem-server 
*** Example code, do not use in production ***
```